src/tools/sdk/dart_sdk_archive/lib/src/version_selector.dart

```diff
@@ -144,13 +144,26 @@ class VersionSelector {
           }
         }
 
+        // No Mac 32-bit SDK builds after 2.80
         if (name == 'macOS' && platformVariant.architecture == 'ia32') {
-          // No Mac 32-bit SDK builds >= 2.80
           if (versionInfo.version > Version(2, 7, 0)) {
             continue;
           }
         }
 
+        // No Mac arm64 SDK builds before 2.14.0-281.0.dev, and not in stable yet.
+        // TODO: After 2.14 simply change these tests to versionInfo.version < Version(2,14,0).
+        if (name == 'macOS' && platformVariant.architecture == 'arm64') {
+          if (versionInfo.version < Version(2, 14, 0, pre: '281.0.dev')) {
+            continue;
+          }
+
+          if (versionInfo.channel == 'stable') {
```

Comment:
Why keep that? For now, it's dead code (`2.13.x` is less than `2.14.0-281.0.dev` so on stable this if is never reached) and when 2.14.0 is released it will be wrong to skip stable (at least according to the TODO above).

Reply:
I don't want to automatically show up in stable until we know for sure that everything works are intended.

Reply:
LGTM then.